tests/rbd/rbd_utils.py

```diff
@@ -919,3 +945,38 @@ def verify_migration_commit(rbd, pool, image):
     else:
         log.error((f"Image {image} is not found in pool {pool}"))
         raise ImageNotFoundError("Image Not found.")
+
+
+def verify_namespace_create(rbd, pool, namespace):
+    """verify namespace creation
+
+    Args:
+        rbd: rbd_object
+        pool: pool name
+    """
+    log.info("verifying namespcae creation")
+    out = rbd.exec_cmd(cmd=f"rbd namespace ls --pool {pool} --format json", output=True)
+    result = json.loads(str(out))
+    for value in result:
+        if value["name"] == namespace:
+            log.info(f"namespace {namespace} is successfully created")
+            return 0
+    log.error(f"namespace {namespace} creation failed")
+    return 1
+
+
+def verify_namespace_image_exit(rbd, pool, namespace, imagename):
+    """Verify image creation at namespace
+
+    Args:
+        rbd: rbd_object
+        pool: name of pool
+        namespace: name of namespace
+        image: name of image
+    """
+    log.info("verifying image creation at namespace")
+    out = rbd.exec_cmd(
+        cmd=f"rbd --pool {pool} --namespace {namespace} ls --format json", output=True
+    )
+    images = json.loads(str(out))
+    return True if any(imagename == image for image in images) else False
```

Comment:
[Similar to above comment]
Did we try modifying exiting method to check whether image exists ?
IMO, having two different method creates confusion.

images in namespaces to be treated as same as other images just apart from the name

Reply:
If we are going to have dedicated methods, please change the method name from exit to exists

Reply:
Done. Accommodate with existing method image_exist().